jwst/pipeline/calwebb_spec2.py

```diff
@@ -220,15 +221,17 @@ def process_exposure_product(
         # Technically there should be just one.
         # We'll just get the first one found
         imprint = members_by_type['imprint']
-        if exp_type in ['NRS_MSASPEC', 'NRS_IFU'] and \
-           len(imprint) > 0:
+        if len(imprint) > 0 and (exp_type in ['NRS_MSASPEC', 'NRS_IFU'] or \
+           is_nrs_ifu_flatlamp(input)):
```

Comment:
Do we expect anyone to ever take associated imprint (light leak) exposures along with lamp exposures?

Reply:
That's up to the association rules.  According to the INS requirements table, imprint_subtract is supposed to be run for NRS_LAMP exposures with lamp_mode = 'IFU', although they have a footnote in that says "leakage exposures not specifiable as such in lamp template, so no automatic association?"